SyntheticsRunTestsTask/package.json

```diff
@@ -36,13 +40,14 @@
   "devDependencies": {
     "@types/deep-extend": "^0.4.32",
     "@types/jest": "^27.0.2",
-    "@types/node": "^10.17.0",
+    "@types/node": "^16.18.0",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
     "eslint": "^7.32.0",
     "eslint-plugin-jest": "^24.4.2",
     "eslint-plugin-prettier": "4.0.0",
     "jest": "^27.3.1",
+    "node-gyp": "^9.3.1",
```

Comment:
Why do we need node-gyp installed?

Reply:
At some point, I thought the `node-gyp` compilation errors were making the CI fail, but it was actually `yarn install --production`. I'll see if it works without it, my bad.